test/WebSites/WebApi/EndPoints/OpenApiEndpoints.cs

```diff
@@ -42,10 +42,18 @@ public static IEndpointRouteBuilder MapWithOpenApiEndpoints(this IEndpointRouteB
             })
             .WithOpenApi();
 
-            group.MapPost("/IFromFile", (IFormFile file) =>
+            group.MapPost("/IFromFile", (IFormFile file, string queryParameter) =>
             {
-                return file.FileName;
-            }).WithOpenApi();
+                return $"{file.FileName}{queryParameter}";
+            }).WithOpenApi(o =>
+            {
+                var parameter = o.Parameters.FirstOrDefault(p => p.Name.Equals("queryParameter", StringComparison.InvariantCulture));
```

Comment:
```suggestion
                var parameter = o.Parameters.FirstOrDefault(p => p.Name.Equals("queryParameter", StringComparison.OrdinalIgnoreCase));
```

Could you also change this one too please?

https://github.com/domaindrivendev/Swashbuckle.AspNetCore/blob/17981d9626ce04cc48932aa7443f909653d5fcdd/test/WebSites/Basic/Controllers/FilesController.cs#L44